eda/src/autogluon/eda/analysis/model.py

```diff
@@ -199,6 +207,31 @@ def _fit(self, state: AnalysisState, args: AnalysisState, **fit_kwargs):
         label = predictor.label
         y_true = val_data[label]
         y_pred = predictor.predict(val_data)
+
+        if predictor.problem_type in [BINARY, MULTICLASS]:
+            y_proba = predictor.predict_proba(val_data)
+            highest_error = y_proba[y_true != y_pred].max(axis=1)
+            highest_error.name = "error"
+
+            scores = np.sort(y_proba.values, axis=1)
+            diff = scores[:, -1] - scores[:, -2]
+            undecided = pd.Series(index=y_pred.index, data=diff, name="score_diff").sort_values(ascending=True)
+            undecided = val_data.join(y_proba).join(undecided).sort_values(by="score_diff")
+            highest_error = (
+                val_data.join(y_proba, rsuffix="_pred")
+                .join(highest_error, how="inner")
+                .sort_values(by="error", ascending=False)
+            )
+        elif problem_type == REGRESSION:
+            highest_error = np.abs(y_pred - y_true).sort_values(ascending=False)
+            highest_error.name = "error"
+            highest_error = (
+                val_data.join(y_pred, rsuffix="_pred")
+                .join(highest_error, how="inner")
+                .sort_values(by="error", ascending=False)
+            )
+            undecided = None
+
```

Comment:
Should we have an `else` here for if problem types isn't `BINARY, MULTICLASS, or REGRESSION`. Could it be softclass or quantile? If so, should we handle that or throw?